### PR TITLE
Update crew

### PR DESCRIPTION
**Pull request by lovedheart**
run remove_la_files before compiling packages to resolve any .la related issues

#2108

### CODE REVIEW

File: crew [hunk 1]
```diff
@@ -407,6 +407,9 @@ end
 def build_and_preconfigure (target_dir)
   Dir.chdir target_dir do
     puts "Building from source, this may take a while..."
+    if ARCH == "x86_64"
+      @pkg.remove_la_files
```
Comment (uberhacker) [inline crew]: Please remove the logic from package.rb and put here instead.  You don't even need to define a function out of it.

Comment (uberhacker) [reply crew]: Where are the instructions to execute `make distclean`?  Isn't that part of the process?

**Comment by uberhacker** [comment]
Nice effort to research this issue @lovedheart!  Please see comments.

**Comment by lovedheart** [comment]
i am doing adjustment

File: crew (hunk 1 above)
Comment (lovedheart) [reply crew]: Don't need it, since we don't perform any "make" process before.

File: crew [hunk 2]
```diff
@@ -407,6 +407,14 @@ end
 def build_and_preconfigure (target_dir)
   Dir.chdir target_dir do
     puts "Building from source, this may take a while..."
+    
+    # Remove /usr/local/lib64/*.la to resolve any "can not find /usr/local/*.la" compile errors when building the packages (only needed for x86_64).
+    # https://gnunet.org/faq-la-files
+    # https://stackoverflow.com/questions/42963653/libquadmath-la-is-not-a-valid-libtool-archive-when-configuring-openmpi-with-g
+    if ARCH == "x86_64"
+      system "rm -rf #{CREW_LIB_PREFIX}/*.la"   # remove all .la files in /usr/local/lib64 to resolve any .la related compile issues.
```
Comment (uberhacker) [inline crew]: Allow for verbose output?
```
if @opt_verbose then
  system "rm -rvf #{CREW_LIB_PREFIX}/*.la"
else
  system "rm -rf #{CREW_LIB_PREFIX}/*.la"
end
```

File: crew [hunk 3]
```diff
@@ -406,7 +406,16 @@ end
 
 def build_and_preconfigure (target_dir)
   Dir.chdir target_dir do
-    puts "Building from source, this may take a while..."
+    puts "Building from source, this may take a while..."    
+    
+    # Remove /usr/local/lib64/*.la to resolve any "can not find /usr/local/*.la" compile errors when building the packages (only needed for x86_64).
+    # https://gnunet.org/faq-la-files
+    # https://stackoverflow.com/questions/42963653/libquadmath-la-is-not-a-valid-libtool-archive-when-configuring-openmpi-with-g
+    if @opt_verbose then
```
Comment (uberhacker) [inline crew]: What happened to the `if ARCH == 'x86_64'` conditional?

Comment (lovedheart) [reply crew]: Only required fo x86_64

Comment (uberhacker) [reply crew]: Right but now it deletes `*.la` files for all architectures.

Comment (lovedheart) [reply crew]: there are not any errors like "*.la" not a valid libtool archiv

File: crew [hunk 4]
```diff
@@ -406,7 +406,20 @@ end
...
+        if ARCH == "x86_64"
+          system "rm -rf #{CREW_LIB_PREFIX}/*.la"   # remove all .la files in /usr/local/lib64 to resolve any .la related compile issues.
+        end 
+    else
+        if ARCH == "x86_64"
+          system "rm -rf #{CREW_LIB_PREFIX}/*.la"   # remove all .la files in /usr/local/lib64 to resolve any .la related compile issues.
+        end
+    end
```
Comment (uberhacker) [inline crew]:
```
if ARCH == 'x86_64'
  if @opt_verbose then
    system "rm -rvf #{CREW_LIB_PREFIX}/*.la"
  else
    system "rm -rf #{CREW_LIB_PREFIX}/*.la"
  end
end
```

Comment (uberhacker) [reply crew]: You only need to add the comment once above the conditional.

**Comment by uberhacker** [comment]
I will put this to the test soon!  Nice work.

**Comment by lovedheart** [comment]
@uberhacker I have tested it to build mesa

**Comment by lovedheart** [comment]
#2032

**Comment by lovedheart** [comment]
Fix #1606

**Pull request merged**